components/kyma-environment-broker/internal/process/provisioning/busola_migrator_overrides.go

```diff
@@ -0,0 +1,38 @@
+package provisioning
+
+import (
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/sirupsen/logrus"
+)
+
+const BusolaMigratorComponentName = "busola-migrator"
+
+type BusolaMigratorOverridesStep struct {
+}
+
+func NewBusolaMigratorOverridesStep() *BusolaMigratorOverridesStep {
+	return &BusolaMigratorOverridesStep{}
+}
+
+func (s *BusolaMigratorOverridesStep) Name() string {
+	return "BusolaMigratorOverrides"
+}
+
+func (s *BusolaMigratorOverridesStep) Run(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
+	kubeconfigIDOverrides := []*gqlschema.ConfigEntryInput{
+		{
+			Key:   "deployment.env.kubeconfigID",
```

Comment:
in the implementation you are setting deployment.env.kubeconfigID, but in the test deployment.env.instanceID and the test is green :(